utilities/test_suite/HOST/Tensor_audio_host.cpp

```diff
@@ -464,5 +469,7 @@ int main(int argc, char **argv)
     free(dstDims);
     free(inputf32);
     free(outputf32);
+    if (testCase == 6)
+        free(window.lookup);
```

Comment:
instead of doing that, it is better to check the pointer validity

Reply:
Done